spec.html

```diff
@@ -33772,7 +33772,7 @@ <h1>Date.prototype.toISOString ( )</h1>
           1. Let _dateObject_ be the *this* value.
           1. Perform ? RequireInternalSlot(_dateObject_, [[DateValue]]).
           1. Let _tv_ be _dateObject_.[[DateValue]].
-          1. If _tv_ is not finite, throw a *RangeError* exception.
+          1. If _tv_ is *NaN*, throw a *RangeError* exception.
```

Comment:
If we were to make this change, I would probably add

```suggestion
          1. If _tv_ is *NaN*, throw a *RangeError* exception.
          1. Assert: _tv_ is an integral Number.
```